lib/espree.js

```diff
@@ -66,11 +66,7 @@ export default () => Parser => {
             // Initialize acorn parser.
             super({
 
-                // TODO: use {...options} when spread is supported(Node.js >= 8.3.0).
-                ecmaVersion: options.ecmaVersion,
-                sourceType: options.sourceType,
-                ranges: options.ranges,
-                locations: options.locations,
+                ...options,
```

Comment:
I think the `TODO` was wrong, and that we should rather remove the comment. This change would most likely allow configuring unsupported options, i.e., configuring the underlying `acorn` through `espree`.

Reply:
maybe (but not sure) I was the one who wrote the TODO. :-( 

for those using the undocumented options, we have 3 choices:

* 1. throw an error.
* 2. passing the option to acorn (and output a warning could be better).
* 3. just ignoring the option (current behavior)

IMHO, 1 ~ 2 > 3.

Reply:
I agree with @mdjermanovic, let's keep current behavior and remove the comment?

Reply:
I agreed, but there is a room to improve here - it could throw an error, rather than ignoring the option silently.

Reply:
I'm not familiar with the options, I'll let you work on it then.

Reply:
I think it makes sense to evaluate throwing an error on unknown options, but that would probably be a breaking change for Espree _and_ ESLint.

As for this issue, the TODO comment, it might be best to replace it with a comment saying the opposite - do not use spread, because we don't want to pass any unknown options to `acorn`.